python/private/flags.bzl

```diff
@@ -21,6 +21,40 @@ unnecessary files when all that are needed are flag definitions.
 load("@bazel_skylib//rules:common_settings.bzl", "BuildSettingInfo")
 load("//python/private:enum.bzl", "enum")
 
+def _FlagEnum_flag_values(self):
+    return sorted(self.__members__.values())
+
+def FlagEnum(**kwargs):
```

Comment:
question: Should `FlagEnum` be also used in some of the `pypi` integration flags?

Reply:
Sure. I just created this because writing that same `sorted(e.members.values())` was looking pretty redundant. This PR was big enough and I didn't want to go update all the stuff in flags.bzl to use it.